client/src/components/Overview/ImageGalleryThumbnail.jsx

```diff
@@ -5,12 +5,15 @@ var ImageGalleryThumbnail = function(props) {
     return (
       <React.Fragment>
         <div className='image-gallery-thumbnail'>
+          <img src={props.thumbnail_url} className="ig-thumbnail" />
           <hr className='three-px-hr'/>
         </div>
       </React.Fragment>
     );
   } else return (
-    <div className='image-gallery-thumbnail'></div>
+    <div className='image-gallery-thumbnail'>
+      <img src={props.thumbnail_url} className="ig-thumbnail"/>
```

Comment:
if you pass in some info about the image, you could add an `alt` tag to these for accessibility

Reply:
I do need to add an alt tag, thanks for the nudge. 
I don't know that I have any useful data to base it on other than the image's index in the array.